src/lu.jl

```diff
@@ -110,6 +112,9 @@ __lu(A::StaticMatrix{M,0,T}, ::Val{Pivot}) where {T,M,Pivot} =
 __lu(A::StaticMatrix{1,1,T}, ::Val{Pivot}) where {T,Pivot} =
     (SMatrix{1,1}(one(T)), A, SVector(1))
 
+__lu(A::LinearAlgebra.HermOrSym{T,<:StaticMatrix{1,1,T}}, ::Val{Pivot}) where {T,Pivot} =
```

Comment:
Out of curiosity, what is the purpose of this additional method? Just a performance optimization?

Reply:
IIRC yes, that's just an optimization similar to `__lu(A::StaticMatrix{1,1,T}, ::Val{Pivot})`.